### PR TITLE
Fix EasyMDE error when input Enter

### PR DESCRIPTION
**Pull request by wxiaoguang**
The PR https://github.com/go-gitea/gitea/pull/18911  removes global `window.CodeMirror`, now when input Enter in EasyMDE, there is an error (`window.CodeMirror.Pass`):

```
EasyMDE.js:80 Uncaught TypeError: Cannot read properties of undefined (reading 'Pass')
    at Enter (EasyMDE.js:80:34)
    at doHandleBinding (easymde.js:8129:14)
    at easymde.js:8200:61
    at lookupKey (easymde.js:7774:26)
    at lookupKeyForEditor (easymde.js:8142:37)
    at dispatchKeyInner (easymde.js:8170:18)
    at dispatchKey (easymde.js:8166:12)
    at handleKeyBinding (easymde.js:8200:14)
    at CodeMirror.onKeyDown (easymde.js:8219:19)
    at HTMLTextAreaElement.<anonymous> (easymde.js:4928:22)
```

This PR fixes the error.

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

File: web_src/js/features/comp/EasyMDE.js [hunk 1]
```diff
@@ -74,10 +74,10 @@ export async function createCommentEasyMDE(textarea, easyMDEOptions = {}) {
   const inputField = easyMDE.codemirror.getInputField();
   inputField.classList.add('js-quick-submit');
   easyMDE.codemirror.setOption('extraKeys', {
-    Enter: () => {
+    Enter: (cm) => {
       const tributeContainer = document.querySelector('.tribute-container');
       if (!tributeContainer || tributeContainer.style.display === 'none') {
-        return window.CodeMirror.Pass;
+        cm.execCommand('newlineAndIndent');
```
Comment (silverwind) [inline web_src/js/features/comp/EasyMDE.js:80]: Why not `cm.Pass`?

Comment (wxiaoguang) [reply web_src/js/features/comp/EasyMDE.js:80]: `CodeMirror.Pass` exists, while `cm = CodeMirror(...); cm.Pass` doesn't exist.

I have read through the EasyMDE and CodeMirror code, it's impossible to access `CodeMirror.Xxx` in Gitea now.

Comment (silverwind) [reply web_src/js/features/comp/EasyMDE.js:80]: Fine I guess if the end result is the same.

Comment (wxiaoguang) [reply web_src/js/features/comp/EasyMDE.js:80]: It is the same. CodeMirror document said: the default command for `Enter` is `newlineAndIndent`.

https://codemirror.net/doc/manual.html

image

**Pull request merged**